srv/SpawnWorld.srv

```diff
@@ -0,0 +1,5 @@
+string world_model                # Entity world model name
+ue_msgs/EntityState state         # World given name + initial pose
```

Comment:
```suggestion
geometry_msgs/Pose pose       # World given name + initial pose
```
geometry_msgs/Pose pose

Reply:
Ok aside from the pose, let it have a separate string world_name also

Reply:
Ok aside from the pose, let it have a separate `string name` also